src/pages/logos.tsx

```diff
@@ -0,0 +1,177 @@
+                {companyName}
+              </h2>
+              <p className="text-white text-center mt-5">{description}</p>
+            </div>
+          </div>
+        </div>
+      )}
+    </div>
+  );
+}
+
+function chunkArray<T>(array: T[], size: number): T[][] {
+  const chunkedArr: T[][] = [];
+  let index = 0;
+  while (index < array.length) {
+    chunkedArr.push(array.slice(index, size + index));
+    index += size;
+  }
+  return chunkedArr;
+}
+function RenderLogos(packageList: any[], rowSize: number, logoSize: string) {
+  return (
+    <>
+      {chunkArray(packageList, rowSize).map((chunk, rowIndex) => (
+        <div
+          key={rowIndex}
+          className="flex flex-row justify-center items-center mb-4 gap-5 mt-[100px]"
+        >
+          {chunk.map((exhibitor, idx) => (
+            <div key={idx}>
+              <Logo
+                pic={exhibitor.logoColor} // exhibitor.logoColor
+                companyName={exhibitor.name}
+                description={exhibitor.description}
+                size={logoSize}
+              />
+            </div>
+          ))}
+        </div>
+      ))}
+    </>
+  );
+}
+type LogosProps = {
+  exhibitorData: {
+    id?: string;
+    name: string;
+    logoWhite?: string | null;
+    logoColor?: string | null;
+    description?: string;
+    package?: string;
+    jobOfferId?: string;
+  }[];
+};
+export default function Logos({ exhibitorData }: LogosProps) {
+  const t = useLocale();
+  const headSponsor = exhibitorData.filter(
+    (e) => e.name.toLowerCase() === "omegappoint"
+  );
+  const premiumPackages = exhibitorData.filter((e) => e.package === "premium");
+  const headhunterPackages = exhibitorData.filter(
+    (e) => e.package === "headhunter"
+  );
+  const sponsorPackages = exhibitorData.filter((e) => e.package === "sponsor");
+  const basePackages = exhibitorData.filter((e) => e.package === "base");
+  const startupPackages = exhibitorData.filter((e) => e.package === "startup");
+
+  return (
+    <div className="pt-[200px] pb-[300px]">
+      <h1 className="uppercase text-cerise text-5xl font-medium text-center">
+        {" "}
+        {t.logos.header}
+      </h1>
+      <div className="block sm:hidden">
+        {RenderLogos(headSponsor, 1, "w-[250px]")}
+        {RenderLogos(premiumPackages, 2, "w-[125px]")}
+        {RenderLogos(headhunterPackages, 2, "w-[125px]")}
+        {RenderLogos(sponsorPackages, 3, "w-[80px]")}
+        {RenderLogos(basePackages, 3, "w-[80px]")}
+        {RenderLogos(startupPackages, 3, "w-[80px]")}
+      </div>
+
+      <div className="hidden sm:block">
+        {RenderLogos(headSponsor, 1, "w-[600px]")}
+        {RenderLogos(premiumPackages, 2, "w-[400px]")}
+        {RenderLogos(headhunterPackages, 2, "w-[400px]")}
+        {RenderLogos(sponsorPackages, 4, "w-[300px]")}
+        {RenderLogos(basePackages, 4, "w-[300px]")}
+        {RenderLogos(startupPackages, 4, "w-[300px]")}
+      </div>
```

Comment:
Think these values can be made smaller. On my screen, each logo takes up a lot of space. Also, another thing to consider is to make all logos a consistently smaller size and change widths instead.

Reply:
Maade the pictures slightly smaller.